Update svm.rst

Naereen opened the pull request:
Just two tiny typos, and space errors in the references.

Naereen committed:
Update svm.rst

Just two tiny typos, and space errors in the references.

nelson-liu commented on doc/modules/svm.rst:
```diff
@@ -705,7 +705,7 @@ computations. These libraries are wrapped using C and Cython.
     - `LIBSVM: a library for Support Vector Machines
       <http://www.csie.ntu.edu.tw/~cjlin/papers/libsvm.pdf>`_
 
-    - `LIBLINEAR -- A Library for Large Linear Classification
```
The name of the paper / project is "LIBLINEAR -- A Library for Large Linear Classification", and should probably be cited as such.

Naereen replied:
True, sorry, I will do a second PR after this one get accepted.

nelson-liu replied:
probably better to commit again, and then squash your commits into one.

Naereen replied:
OK, done.

agramfort commented on doc/modules/svm.rst:
```diff
@@ -611,16 +611,16 @@ holds the support vectors, and ``intercept_`` which holds the independent
-   <http://www.springerlink.com/content/k238jx04hm87j80g/>`_
-   C. Cortes, V. Vapnik, Machine Leaming, 20, 273-297 (1995)
+   <http://www.springerlink.com/content/k238jx04hm87j80g/>`_,
+   C. Cortes, V. Vapnik - Machine Leaming, 20, 273-297 (1995).
```
Leaming -> learning

Naereen committed:
Update svm.rst

Thanks @agramfort.

Naereen committed:
Update cross_validation.rst

Weird display bug on a Warning box on cross_validation.rst
("Contrary to StratifiedKFold, the ``labels`` of :class:`LeaveOneLabelOut` should not encode the target class to predict...")

Naereen commented:
How often does the live documentation gets updated? (branch ``stable``)
The live doc seems to date from 05-11-15, 4 months from now..

GaelVaroquaux commented:
> How often does the live documentation gets updated? (branch stable)

Each time there is a release.

> The live doc seems to date from 05-11-15, 4 months from now..

The dev one should be up to date: http://scikit-learn.org/dev/

GaelVaroquaux commented on doc/modules/svm.rst:
```diff
@@ -611,16 +611,16 @@ holds the support vectors, and ``intercept_`` which holds the independent
 
 .. topic:: References:
-
+,
```
Trailing coma, should be removed.

GaelVaroquaux commented:
Thanks for the fixes. That's all I have in terms of comments.
Before we merge, I think that the trailing coma should be addressed.

Naereen committed:
Update svm.rst

Trailing comma removed, sorry.

Naereen commented:
OK, done.
Sorry about the /stable and /dev confusion, I should enforce my browser to read at the latest version (otherwise I will come again and try to fix issues already fixed!).

agramfort commented:
Thx @Naereen
